### PR TITLE
Fix: WithSecure format

### PR DESCRIPTION
**Pull request by vg-svitla**

### CODE REVIEW

File: WithSecure/withsecure-elements/ingest/parser.yml [hunk 1]
```diff
@@ -73,6 +80,15 @@ stages:
           withsecure.file.rarity: "{{json_event.message.details.rarity}}"
           withsecure.recommended_action: "{{json_event.message.details.recommandedAction}}"
 
+      - set:
+          user.name: "{{json_event.message.userName}}"
+          user.domain: "{{json_event.message.userName}}"
```
Comment (squioc) [inline WithSecure/withsecure-elements/ingest/parser.yml:85]: My bad. Can you split the content of `userName` with the first part for `user.domain` and the second part in `user.name` when `userName` contains `\` or `//`?

File: WithSecure/withsecure-elements/tests/edr_event.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      "id": "d232eaa8-8e11-4574-96b6-b3b406c03c75",
+      "name": "Example"
+    },
+    "related": {
+      "hosts": ["MyComputer"],
+      "user": ["MyComputer/john.doe"]
+    },
+    "user": {
+      "domain": "MyComputer/john.doe",
+      "name": "MyComputer/john.doe"
```
Comment (squioc) [inline WithSecure/withsecure-elements/tests/edr_event.json:40]:
```suggestion
      "domain": "MyComputer",
      "name": "john.doe"
```

File: WithSecure/withsecure-elements/tests/edr_event.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      "hostname": "MyComputer",
+      "name": "MyComputer"
+    },
+    "organization": {
+      "id": "d232eaa8-8e11-4574-96b6-b3b406c03c75",
+      "name": "Example"
+    },
+    "related": {
+      "hosts": ["MyComputer"],
+      "user": ["MyComputer/john.doe"]
```
Comment (squioc) [inline WithSecure/withsecure-elements/tests/edr_event.json:36]:
```suggestion
      "user": ["john.doe"]
```

File: WithSecure/withsecure-elements/tests/system_event.json [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    "sekoiaio": {
+      "server": {
+        "os": {
+          "type": "windows"
+        }
+      }
+    },
+    "user": {
+      "domain": "MyComputer\\john.doe",
+      "name": "MyComputer\\john.doe"
```
Comment (squioc) [inline WithSecure/withsecure-elements/tests/system_event.json:59]:
```suggestion
      "domain": "MyComputer",
      "name": "john.doe"
```

File: WithSecure/withsecure-elements/tests/system_event.json [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    "organization": {
+      "id": "d232eaa8-8e11-4574-96b6-b3b406c03c75",
+      "name": "Example"
+    },
+    "process": {
+      "pid": 652
+    },
+    "related": {
+      "hosts": ["MyComputer"],
+      "user": ["MyComputer\\john.doe"]
```
Comment (squioc) [inline WithSecure/withsecure-elements/tests/system_event.json:48]:
```suggestion
      "user": ["john.doe"]
```

**Review by squioc — CHANGES_REQUESTED** [review]
Thanks for the work.
Somme changes to apply.

File: WithSecure/withsecure-elements/ingest/parser.yml (hunk 1 above)
Comment (vg-svitla) [reply WithSecure/withsecure-elements/ingest/parser.yml:85]: Fixed thank you!

**Review by squioc — APPROVED** [review]

**Pull request merged**